scintilla/gtk/ScintillaGTK.cxx

```diff
@@ -3250,6 +3250,7 @@ void scintilla_release_resources(void) {
 static void *copy_(void *src) { return src; }
 static void free_(void *doc) { }
 
+GEANY_API_SYMBOL
 GType scnotification_get_type(void) {
 	static gsize type_id = 0;
```

Comment:
>     Anyway the discussion out of scope for this PR
>
> Code review is out of scope for this PR? At the least it could do like
> the code @b4n <https://github.com/b4n> linked to, and avoid ugly (and
> useless) casts.
>

Yes, reviewing *this* code is out of scope for *this* PR. The code you 
question is not changed in this PR and is already part Geany.

This PR just adds the GEANY_API_SYMBOL decoration and the implementation 
is irrelevant to the PR.